apps.js

```diff
@@ -76,6 +78,8 @@ function createApp(type, addRoutes) {
     stream: logger.stream
   }));
   app.use(middleware.requireHttps(cfg));
+  app.use(connectTimeout(CONST.HTTP_TIMEOUT.TIMEOUT_DURATION));
+  app.use(middleware.timeout());
```

Comment:
Not required. Do the timeout required processing in middleware.error.  By default the connect-timeout module returns HTTP Status 503 - which means service is unavailable temporarily (due to load or maintenance). So it is debatable whether we want to add a new HTTP error GatewayTimeout or use the default one from timeout module.

Reply:
yes.. we can use the default error sent by using `respond` parameter. Need to test once, if it works the same way as Gateway Timeout error. Should work either way.